### DISCUSSION TITLE
Is the object still there?

### DISCUSSION THREAD

**Discussion by jonnymccullagh** [streetcomplete/StreetComplete#4531] [Ideas]
Would be possible to get a quest "Is this still here?" for emergency equipment like defibrillators and fire extinguishers, please?

**Comment by mnalis** [comment]
- it is already being asked for [`emergency=defibrillator`](https://wiki.openstreetmap.org/wiki/Tag:emergency=defibrillator) which are located **outside**:

  https://github.com/streetcomplete/StreetComplete/blob/a84697564f5fae0ed06f601d161c06cc0c5b94ef/app/src/main/java/de/westnordost/streetcomplete/quests/existence/CheckExistence.kt#L44

  It seems it is intentionally (a2d9b8c9bf7046825a5b33f4d3c712b55723ee7a) **not** being asked for those located **inside**, probably due to concern that if indoors they are hard to find, so resurveyors are likely to remove the AED even if it stil exists (i.e. the danger of wrongly updating the map would be higher than the potential benefit).

- The same resurvey could be added for [emergency=fire_extinguisher](https://wiki.openstreetmap.org/wiki/Tag:emergency=fire_extinguisher) which are located **outside**, but there seems to be only `649` of them [mapped worldwide](http://overpass-turbo.eu/s/1mUL), so I'm not sure if that would be worth the effort.

**Reply by mnalis** [reply]
- also ref: https://github.com/streetcomplete/StreetComplete/issues/6092

**Comment by amandasaurus** [comment]
If I'm reading the code, it only asks for a recheck every 4 years. I think every 6 months is better, due to the benefit of saving lives compared to the inconvenience for OSM mappers.

**Reply by mnalis** [reply]
Yes, it seems `4 years` is the default for that quest. StreetComplete users however can in their Preferences use that resurvey period, or **_halve_** it, or **_double_** it.

Feel free to open a [PR](https://github.com/streetcomplete/StreetComplete/pulls) for that, with your rationale in comment - it sounds relatively easy thing to do. 

If the powers-that-be disagree, they'll just reject the PR (hopefully with comment why), and if they agree, they'll just have to do one click to accept it. Thus we can help reducing their workload  - see https://github.com/streetcomplete/StreetComplete/discussions/3450
